modules/ROOT/pages/feature-flagging.adoc

```diff
@@ -0,0 +1,162 @@
+For example, you can configure a Mule 4.3.0 instance to enable and disable specific features for all applications running on this instance:
+
+.Example `wrapper.conf` file
+----
+…
+wrapper.java.additional.999=-DHONOUR_RESERVED_PROPERTIES_PROPERTY=true
+wrapper.java.additional.1000=-DCOMPUTE_CONNECTION_ERRORS_IN_STATS_PROPERTY=false
+----
+
+== Feature Flags Reference
+
+//Confirm this statement which is not clear yet.
+The following table shows the available feature flags, a description of their functionality, the Mule version in which the flag became available, the earlier minor version releases that enable the flag by default, and the issue ID where the change was implemented:
+
+[%header%autowidth.stretch,cols="a,a,a"]
+|===
+|Feature Flag | Description | Issue ID
+<.^|`HONOUR_RESERVED_PROPERTIES_PROPERTY`
+|When enabled, reserved properties such as `app.name` can't be overridden by global properties (overrides are ignored).
+
+*Available Since*
+
+* 4.4.0
+* 4.3.0-202102
+
+*Enabled by Default Since*
+
+* 4.4.0
+* 4.3.1
+<.^| `MULE-17659`
+<.^|`HANDLE_SPLITTER_EXCEPTION_PROPERTY`
+|When enabled, DataWeave handles splitter exceptions and prevents Scala serialization issues.
+
+*Available Since*
+
+* 4.4.0
+
+*Enabled by Default Since*
+
+* 4.4.0
+<.^|`DW-383`
+!===
+<.^|`BATCH_FIXED_AGGREGATOR_TRANSACTION_RECORD_BUFFER_PROPERTY`
+|When enabled, batch aggregators with fixed size aggregators commit only when a full block is processed. See EE-7443 for more information.
+
+*Available Since*
+
+* 4.4.0
+* 4.3.0-202103
+* 4.2.2-202103
+
+*Enabled by Default Since*
+
+* Not enabled by default in any Mule version
+<.^| `MULE-19218`
+<.^|`COMPUTE_CONNECTION_ERRORS_IN_STATS_PROPERTY`
+|When enabled, Connection errors are computed as part of alerts triggering.
+
+*Available Since*
+
+* 4.4.0
+* 4.3.0-202103
+
+*Enabled by Default Since*
+
+* 4.4.0
+* 4.3.1
+<.^| `MULE-19020`
+<.^|`TO_STRING_TRANSFORMER_TRANSFORM_ITERATOR_ELEMENTS_PROPERTY`
+|When enabled, managed cursor iterators transformed to Strings show the representation of the elements instead of generic value `org.mule.runtime.core.internal.streaming.object.-ManagedCursorIteratorProvider$ManagedCursorIterator@######`.
+
+*Available Since*
+
+* 4.4.0
+
+*Enabled by Default Since*
+
+* 4.4.0
+<.^| `MULE-19323`
+<.^|`ENABLE_POLICY_ISOLATION_PROPERTY`
+|When enabled, Mule manages extensions imported by a policy in complete isolation from the extensions imported by the Mule application. Also, validations prevent the usage of explicit configurations that the application declared as part of the policy initialization.
+
+*Available Since*
+
+* 4.4.0
+* 4.3.0-202107
+
+*Enabled by Default Since*
+
+* 4.4.0
+<.^| `MULE-19226`
+<.^|`ENTITY_RESOLVER_FAIL_ON_FIRST_ERROR_PROPERTY`
+|When enabled, the Mule XML DSL parser fails when deploying an application that declares a schema that cannot be located. Otherwise, the parser fails if the application also makes use of the namespace that such schema is bound to.
```

Comment:
the namespace that such schema is bound to. -> the namespace to which such a schema is bound.